_posts/2018-01-10-C4Q-QandA-two.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: epic
+title: "C4Q&A: 2"
+date: 2018-01-10
+author: [orta, sarahweir, roop, alan, chungyi, yuki]
+categories: [culture, teaching, event, panel]
+---
+
+[C4Q is](https://www.c4q.nyc) a non-for-profit hacker school based in NYC. We've had members of Artsy team help out by being TAs, run 
+committees and steer the curriculum as Engineers in the industry for many years. In 2017 C4Q reached out to arrange a meetup
+between half of their students and Artsy. This year we ran a panel for the other half of the C4Q students who study iOS and Web.
```

Comment:
For this one, maybe something like this:

> C4Q is a non-for-profit hacker school based in NYC. We've had members of the Artsy team help out by being TAs, running committees and steering the curriculum as Engineers in the industry for many years. In 2017, C4Q reached out to arrange a meetup between Artsy engineers and students who were learning Android and Web development. This year we ran a panel for the other half of the C4Q students who study iOS and Web.

Here I think the distinction between which half you've got makes more sense.